src/eth_provider/database/types/transaction.rs

```diff
@@ -168,6 +172,7 @@ impl<'a> StoredTransaction {
 
         let transaction = Transaction {
             hash,
+            from: Address::arbitrary(u)?,
```

Comment:
you should probably extract `from` from the random Private key

Reply:
done, thanks